models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Float, ForeignKey, Date
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+class Department(Base):
+    __tablename__ = 'departments'
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String(100), nullable=False)
+    employees = relationship("Employee", back_populates="department")
+
+class Employee(Base):
+    __tablename__ = 'employees'
+
+    id = Column(Integer, primary_key=True)
+    first_name = Column(String(100), nullable=False)
+    last_name = Column(String(100), nullable=False)
+    salary = Column(Float, nullable=False)
+    department_id = Column(Integer, ForeignKey('departments.id'))
+    department = relationship("Department", back_populates="employees")
+    hire_date = Column(Date)
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider making hire_date non-nullable</b></div>
<div id="fix">

The `hire_date` field is nullable by default. Consider if this should be `nullable=False` to ensure all employees have a hire date.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23,1 +23,1 @@
 -    hire_date = Column(Date)
 +    hire_date = Column(Date, nullable=False)
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/12#issuecomment-2514877662>#a8cb59</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged